app/models/katello/concerns/content_facet_host_extensions.rb

```diff
@@ -53,6 +53,13 @@ def find_by_installable_errata(_key, operator, value)
             { :conditions => "#{::Host::Managed.table_name}.id IN (#{facets.pluck(:host_id).join(',')})" }
           end
         end
+
+        def in_content_view_environment(content_view = nil, lifecycle_environment = nil)
```

Comment:
So:

````
Host.in_content_view_environment
```

Returns all hosts that have a content view environment? 

Does the following return all hosts with a content view environment in environment 1?

```
Host.in_content_view_environment(nil, 1)
```